Majasdarbs/03-majasdarbs/03-katrina-tiltina/index.html

```diff
@@ -0,0 +1,213 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/Majasdarbs/03-majasdarbs/03-katrina-tiltina/style.css">
+    <title>Document</title>
+</head>
+<body>
+  <header>
+    <nav>
+      <label class="burger" for="toggle">&#9776;</label>
+      <input type="checkbox" id="toggle">
+      <label class="logo">Katrina</label>
+      <ul>
+          <li><a class="active" href="#work-experience">Work experience</a></li>
+          <li><a href="#skills">Skills</a></li>
+          <li><a href="#languages">Languages</a></li>
+          <li><a href="#education">Education</a></li>
+          <li><a href="#fqa">FAQ</a></li>
+      </ul>
+  </nav>
+</header>
+<main>
+  <div class="main">
+    <h2>Find yourslef an indivudual 
+      solution in web development.</h2>
+    <a class="cta" href="#contactme"><button>Contact</button></a>
+  </div>
+</main>
+
+        
+          <!-- ABOUT ME -->
+          <section id="aboutme">
+              <div>
+              <img src="/Majasdarbs/03-majasdarbs/03-katrina-tiltina/ARM_3355.jpg" alt="profileImg" width="150px" height="100px">
+              </div>
+              <div>
+              <address>
+                  <h1>Katrina Tiltina | Frontend developer</h1>
+                  <a href="tel:+3712385124">+371 28385124</a><br>
+                  <a href="mailto:katrina.tiltin@gmail.com">katrina.tiltin@gmail.com</a><br>
+                  <a href="#contactme">contact me</a>
+              </address>
+              </div>
+          </section>
+          <!-- /ABOUT ME -->
+  
+          <hr>
+          <!-- WORK EXPERIENCE -->
+          <section id="work-experience">
+              <h2>Work experience</h2>
+  
+              <h3>Public Relations Specialist,
+                Cadet Force Centre, The Ministry of Defence of Latvia.</h3>
+              <em>August 2018 – present</em>
+  
+              <ul>
+                  <li>developing and executing communication strategy
+                    arranging press conferences, interviews, and other media and
+                    event appearances.</li>
+                  <li>managing media requests in a timely and professional manner
+                    creating and posting content for institution's social media
+                    accounts.
+                    
+                    </li>
+                  <li>building relationships with new media contacts and maintain
+                    relationships with existing contacts.</li>
+                  <li>creating marketing and promotional materials, both print and
+                    electronic.</li>
+                  <li>researching media coverage and industry trends.</li>
+                  <li>implementing and maintaining institution's website.</li>
+                  <li>providing support on specific projects.</li>
+
+              </ul>
```

Comment:
Not sure, if you need to put every line into li.
Use <p> or just write everything in one <li>